config/enterprise_versions.yml

```diff
@@ -2,7 +2,7 @@ title: v3.5.0
 components:
   cnx-manager:
     image: tigera/cnx-manager
-    version: v3.5.0
+    version: v3.5.1-1
```

Comment:
Should this say `tesla-`?

Reply:
The gen-versions includes an update to add on the `tesla-`. That way when we merge in the future this branch will have the tag updated to match.

Reply:
That file is used to generate a go file that is actually used, it is in the generation where the `tesla-` is added.